client/src/app/util/EmailVerification.js

```diff
@@ -0,0 +1,21 @@
+import emailjs, {init} from '@emailjs/browser';
+init('5A9OSqglcYcpKAMgd');
+
+export const VerifyEmail = (user) => {
+  const templateParams = {
+    to_email: user.useremail,
+    message: `http://localhost:3000/verify/${user.token}`,
```

Comment:
do we want to make this url dynamic so that when we switch between dev and prod versions, we don't need to manually change it?

Reply:
yeah probably, I didnt think about that